test/integration/auth/node_test.go

```diff
@@ -248,6 +275,9 @@ func TestNodeAuthorizer(t *testing.T) {
 	expectAllowed(t, createNode2(node2Client))
 	expectAllowed(t, updateNode2Status(node2Client))
 	expectAllowed(t, deleteNode2(node2Client))
+	// node evicting pod bound to itself is allowed
+	expectAllowed(t, createNode2MirrorPod(node2Client))
+	expectAllowed(t, createNode2MirrorPodEviction(node2Client))
```

Comment:
> at the end of this block of tests, the node2 mirror pod is being evicted. I'd expect all the tests below trying to delete/evict the node2 mirror pod to get NotFound errors, not Forbidden errors.

According to the [docs](https://kubernetes.io/docs/tasks/administer-cluster/static-pod/#behavior-of-static-pods) you cannot delete a mirror pod with the API server, so I expect eviction to work the same way.